logicsim/templates/logicsim/UserInput.html

```diff
@@ -22,6 +22,10 @@
   <label for="input2">Input 2: </label>
   <input type="number" id="input2" name="input2"><br><br>
 
+  <label for="SS1">Start State 1: </label>
```

Comment:
This section of code should be in index.html.
.

Reply:
fixed